lib/command/run-multiple/collection.js

```diff
@@ -138,8 +138,8 @@ class Collection {
    * If value of property `browser` does not match the slected `browser` in conjugation
    * with the selectedRunName then the suite is removed from configuration.
    */
-  filterSelectedBrowsers(selectedRuns, config) {
-    selectedRuns.forEach((selectedRun) => {
+  filterSelectedBrowsers(selectedRuns) {
+    selectedRuns.forEach(() => {
       selectedRuns.forEach((selectedRun) => {
```

Comment:
We really need to the same loop on loop? =)

Reply:
i did not implement on logic of this function